spanner/api/Spanner/CreateBackup.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.Spanner.Admin.Database.V1;
+using Google.Cloud.Spanner.Common.V1;
+using Google.LongRunning;
+using Google.Protobuf.WellKnownTypes;
+using log4net;
+using System;
+using static GoogleCloudSamples.Spanner.Program;
```

Comment:
Would it be possible to include the imports within the tags so that it's a complete example? How is this done for separated samples in other languages?

Reply:
We need refactor all the samples with tags as it's done in [storage](https://github.com/GoogleCloudPlatform/dotnet-docs-samples/pull/973#pullrequestreview-380738531).

Reply:
Done!